StRoot/RTS/src/DAQ_TPC23/itpc23.cxx

```diff
@@ -1371,6 +1391,8 @@ itpc23::itpc23()
 
 	fmt = 0 ;
 
+	fee_words = 0 ;
```

Comment:
This initialization is for me the critical part as an uninitialized `fee_words` was causing random issues for reconstructing iTPC data.